File: JavaScript/8kyu/uniTotal.js
// ASCII Total
// https://www.codewars.com/kata/572b6b2772a38bc1e700007a

// My code
function uniTotal(s) {
  let sum = 0
  for (i of s.split('')){
    sum += i.charCodeAt()
  }
  return sum
}

// Best code
// function uniTotal(str) {
//   var count = 0;
//   for (var i = 0; i < str.length; i++) {
//     count += str.charCodeAt(i);
//   }
//   return count;
// }

File: Python/8kyu/uni_total.py
# ASCII Total
# https://www.codewars.com/kata/572b6b2772a38bc1e700007a

# My code
def uni_total(s):
    sum = 0
    for i in range (len(s)):
        sum = sum + ord(s[i])
    return sum

# Best code
# def uni_total(string):
#     return sum(map(ord, string))